### DISCUSSION TITLE
cannot run model without residual error

### DISCUSSION THREAD

**Discussion by JonKnights** [nlmixrdevelopment/nlmixr#621] [Q&A]
I have a model that runs SAEM with an additive residual error model. If I comment out the add.err term from the ini block and comment out the IPRED ~ add(add.err) line I get the below error:

> Error in parse(text = paste(x, collapse = "\n")) : 
  <text>:2:2: unexpected ')'
1: function(){
2: })
    ^

How can I run this model without a residual error model? Apologies I cannot provide code snippets.

**Comment by mattfidler** [comment]
Hi @JonKnights 

I am not sure what you are trying to do.  

If you are trying to simulate from the model, that is (or will be) supported in `rxode2`, but not in the current `RxODE` and `nlmixr`.  

If you are trying to estimate, this is not supported either.  `nlmixr2` supports fixing the additive error to a very small value which could be sufficient, but as far as I know this is not possible with `nlmixr`

**Reply by JonKnights** [reply]
Thanks @mattfidler !! Trying to estimate. We were able to fix an focei run with very small value but when I tried to run with SAEM I got an error that said cannot fix error to zero. I can reproduce if you've never seen this before.

**Reply by mattfidler** [reply]
Ok.  Well `saem` in `nlmixr` simply cannot fix the residual error parameters to zero (or any other value).  It is unsupported.  I have seen it but I cannot do anything about it.  This was a new feature in `nlmixr2`

**Reply by JonKnights** [reply]
Thanks for the clarification Matt.
